.github/workflows/ms.analysisservices.servers.yml

```diff
@@ -117,7 +117,7 @@ jobs:
   ###############
   job_publish_module:
     name: 'Publish module'
-    if: contains(fromJson('["refs/heads/main", "refs/heads/master"]'), github.ref)
+    if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/master' || github.event.inputs.prerelease == 'true'
```

Comment:
Beware, the flag does not yet exist in main. I'd propose to merge this without the prerelease flag to main

Reply:
The prerelease condition can then be incorporated in @MariusStorhaug s PR

Reply:
Adjusted for this PR, left in #744.

Reply:
Ready for approval @MrMCake

Reply:
Adjusted for this PR, left in #744. 
Ready for approval @MrMCake.